workflows/cmgg-structural.nf

```diff
@@ -132,9 +133,28 @@ workflow CMGGSTRUCTURAL {
     //
 
     SamplesheetConversion.convert(ch_input, file("${projectDir}/assets/schema_input.json"))
-        .multiMap({ meta, cram, crai, bed, oed ->
-            bed: [ meta, bed ]
-            crams: [ meta, cram, crai ]
+        .map { meta, cram, crai, bed, ped ->
+            new_meta = meta + [family:meta.family ?: meta.id]
+            [ new_meta, cram, crai, bed, ped ]
```

Comment:
This part is not entirely clear to me...

Reply:
It checks if there is a family specified for the current sample. If there is a family it will use the family as the `meta.family`. If there is no family it will use the sample name as `meta.family` (which will always be one sample per 'family')